sdk/metric/minmaxsumcount_stress_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This test is too large for the race detector.  This SDK uses no locks
+// that the race detector would help with, anyway.
+// +build !race
+
+package metric_test
+
+import (
+	"context"
+	"math/rand"
+	"testing"
+	"time"
+
+	"go.opentelemetry.io/otel/api/core"
+	"go.opentelemetry.io/otel/sdk/export/metric"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/minmaxsumcount"
+)
+
+func TestStressInt64MinMaxSumCount(t *testing.T) {
+	desc := metric.NewDescriptor("some_metric", metric.MeasureKind, nil, "", "", core.Int64NumberKind)
+	mmsc := minmaxsumcount.New(desc)
+
+	go func() {
```

Comment:
This is going to leak. Probably want to include an end signal. Given the context you need to pass to `Update` you could pass an external context in with a cancel signal.

Not the end of the world given the short life of the test harness, but something to consider.

Reply:
@paivagustavo looping for a definite amount of time could work, but I like returning with the signal included in the context. The context is already going to be there and it doesn't waste any more computation time than needed.

Though, again, doesn't need to be the most resource conservative given where this lives.

Reply:
I used the `context.WithCancel(..)` idiom and `defer cancelFunc()`.